### DISCUSSION TITLE
Can MIDI2LR receive and map sysex messages from a controller?

### DISCUSSION THREAD

**Discussion by CaptainCathode** [rsjaffe/MIDI2LR#609] [Q&A]
I'm new to MIDI2LR and looking to repurpose my [Korg nanoKONTROL Studio](https://www.korg.com/nz/products/computergear/nanokontrol_studio/) to use with LR.

The nanoKONTROL Studio supports 5 separate "Scenes" which allow each control to be individually assigned to multiple switchable MIDI channels at the press of the Scene button. For example, the same button sends CC12 on MIDI channels 1 through 5 depending on which Scene is active. 

MIDI2LR sees and displays the correct channel/CC/value combination for each button push, but doesn't see that the Scene button has been pressed to switch MIDI channels. I was hoping to be able to use the controller's Scene button to have MIDI2LR switch LR modules when the scene is changed.

I know I can achieve a similar outcome by having the MIDI2LR app switch between multiple config files but wondering if there's another option for MIDI2LR to recognise the channel switch?

Using a MIDI monitor app I can see the button press sends a 12-byte sysex message indicating which scene has been selected. 
Is there any way to configure MIDI2LR to see and act on a sysex message (either in the current version or future planned feature request)? I can't see this in the docs so asking in the forum.

BTW congrats on developing such a great app - donation inbound

**Comment by rsjaffe** [comment]
MIDI2LR ignores sysex messages. There's too much variability among controllers.
